### ISSUE TITLE
Sidekiq 6.5 Compatibility issue (should be a quick fix)

### ISSUE THREAD

**Issue by x9sim9** [projectivetech/sidekiq-queue-pause#3]
in /gems/ruby-2.7.5/gems/sidekiq-6.5.8/lib/sidekiq/fetch.rb   
sidekiq have introduced this line

34:     if @strictly_ordered_queues
35:        @queues.uniq!
36:        @queues << {timeout: TIMEOUT}
37:     end

This is causing 
in /gems/sidekiq-queue-pause-0.1.0/lib/sidekiq-queue-pause.rb
71:    Sidekiq::QueuePause.paused?(q.gsub('queue:', ''), Sidekiq::QueuePause.process_key)

because the queues array now has a hash as the last item of the queue

Any chance this could be fixed?

**Comment by x9sim9** [comment]
```
        if q.class.name == "String"
            q != Sidekiq::BasicFetch::TIMEOUT &&
              Sidekiq::QueuePause.paused?(q.gsub('queue:', ''), Sidekiq::QueuePause.process_key)
          end
        end
```

Seems to fix the issue

**Comment by bigzed** [comment]
First of all, sorry for the long delay and thank you for the issue. It is fixed and released as `0.1.1`.
